### ISSUE TITLE
The Batch Normalization in deploy.prototxt?

### ISSUE THREAD

**Issue by ly-atdawn** [camel007/caffe_seetaFace_Identification#8]
In deploy.prototxt, the layer of  BatchNorm is our common Batch Normalization or FNL?
If it is the first one, could you put your test results in here? 
Thanks!
